enterprise_knowledge_retriever/README.md

```diff
@@ -87,60 +93,96 @@ To perform this setup, you must be a SambaNova customer with a SambaStudio accou
 
 NOTE: Be sure to set batch size model parameter to 32.
 
-1. Update API information for the SambaNova embedding endpoint in the **`sn-ai-starter-kit/.env`** file in the root repo directory. For example: 
+1. Update API information for the SambaNova embedding endpoint in the **`sn-ai-starter-kit/.env`** file in the root repo directory. For example:
+
+    - Assume you have an endpoint with the URL
+        "https://api-stage.sambanova.net/api/predict/nlp/12345678-9abc-def0-1234-56789abcdef0/456789ab-cdef-0123-4567-89abcdef0123"
+    - You can enter the following in the env file (with no spaces):
+
+        ```bash
+            SAMBASTUDIO_EMBEDDINGS_BASE_URL="https://api-stage.sambanova.net"
+            SAMBASTUDIO_EMBEDDINGS_PROJECT_ID="12345678-9abc-def0-1234-56789abcdef0"
+            SAMBASTUDIO_EMBEDDINGS_ENDPOINT_ID="456789ab-cdef-0123-4567-89abcdef0123"
+            SAMBASTUDIO_EMBEDDINGS_API_KEY="89abcdef-0123-4567-89ab-cdef01234567"
+        ```
 
-* Assume you have an endpoint with the URL
-    "https://api-stage.sambanova.net/api/predict/nlp/12345678-9abc-def0-1234-56789abcdef0/456789ab-cdef-0123-4567-89abcdef0123"
-* You can enter the following in the env file (with no spaces):
-    ```
-    EMBED_BASE_URL="https://api-stage.sambanova.net"
-    EMBED_PROJECT_ID="12345678-9abc-def0-1234-56789abcdef0"
-    EMBED_ENDPOINT_ID="456789ab-cdef-0123-4567-89abcdef0123"
-    EMBED_API_KEY="89abcdef-0123-4567-89ab-cdef01234567"
-    ```
 2. In the [config file](./config.yaml), set the variable `embedding_model` to `"sambastudio"`
 
-  > NOTE: Using different embedding models (cpu or sambastudio) may change the results, and change How the embedding model is set and what the parameters are. 
-  > 
-  > You can see the difference in how they are set in the [vectordb.py file](../vectordb/vector_db.py)  (`load_embedding_model method`).
+    > NOTE: Using different embedding models (cpu or sambastudio) may change the results, and change How the embedding model is set and what the parameters are. 
+    > 
+    > You can see the difference in how they are set in the [vectordb.py file](../vectordb/vector_db.py)  (`load_embedding_model method`).
 
 ### Install system dependencies
 
-   - Ubuntu instalation:
-      ```
-      sudo apt install tesseract-ocr
-      ```
-   - Mac Homebrew instalation:
-      ```
-      brew install tesseract
-      ```
-  - Windows instalation:
-      > [Windows tessearc instalation](https://github.com/UB-Mannheim/tesseract/wiki)
+- Ubuntu installation:
 
-   - For other linux distributions, follow the [**Tesseract-OCR installation guide**](https://tesseract-ocr.github.io/tessdoc/Installation.html) 
+    ```bash
+    sudo apt install tesseract-ocr
+    ```
+
+- Mac Homebrew installation:
+
+    ```bash
+    brew install tesseract
+    ```
+
+- Windows installation:
+    > [Windows tessearc instalation](https://github.com/UB-Mannheim/tesseract/wiki)
```

Comment:
instalation should be installation.

This is in a few places so just find replace.